nncf/experimental/openvino_native/quantization/quantize.py

```diff
@@ -34,6 +34,24 @@
 from nncf.common.utils.timer import timer
 
 
+def dump_parameters(model: ov.Model, parameters: Dict, path: Optional[List] = []) -> None:
+    """
+    Dumps input parameters into Model's meta section.
+
+    :param model: ov.Model instance.
+    :param parameters: Incoming dictionary with parameters to save.
+    :param path: Optional list of the paths.
+    """
+    for key, value in parameters.items():
+        # Special condition for composed fields like IgnoredScope
+        if isinstance(value, IgnoredScope):
+            dump_parameters(model, value.__dict__, [key])
+            continue
+        value = value if value else None
+        rt_path = ['optimization'] + path + [key]
+        model.set_rt_info(value, rt_path)
```

Comment:
this `.set_rt_info` is quite limited in terms of the ability to represent and save objects of various types. There can be problems with ignored scope (exception). You should test it.

Reply:
Added to string conversion & test for this.